package.json

```diff
@@ -109,7 +109,8 @@
     "ts-node": "10.0.0",
     "tsconfig-paths": "^3.9.0",
     "typechain": "5.1.1",
-    "typescript": "4.5.2"
+    "typescript": "4.5.2",
+    "web3-utils": "^4.0.5"
```

Comment:
blz explicit version 🙏

Reply:
Sure 😁 

Should be ok now ✅

Reply:
getting this error on gh tests, any clue?
```
error @noble/hashes@1.3.1: The engine "node" is incompatible with this module. Expected version ">= 16". Got "14.21.3"
```

Reply:
Huh, let me try them locally

Reply:
I'm getting no issues on my end (I'm using yarn `1.22.19` and node `v16.15.0`)